clients/privacy-center/README.md

```diff
@@ -19,6 +19,43 @@ Additionally, if you're handy with React and are feeling a little brave you can
 customize the entire application by modifying the TypeScript source code in
 `pages/index.tsx` and `components/RequestModal.tsx`.
 
+### Configuring CSS
+
+In order to modify the way your application appears visually, you can add custom
+css to the `config/config.css` file. For example, to change the application's
+font to Courier:
+
+```css
+* {
+  font-family: Courier !important;
+}
+```
+
+Additionally, because this application uses CSS variables, you can modify those
+CSS variables directly, rather than adding custom CSS.
+
+The only caveat to this is that we need to do so in a way that overrides the CSS
+variables' original values so that we can avoid the complexity of modifying the
+JavaScript file that holds the base values.
+
+Our recommendation is that you do so by using selector specificity, as in the
+following example:
+
+```css
+:root:root {
+  /* Changes the text color to red */
+  --chakra-colors-gray-600: #f00;
+
+  /* Changes the background color to blue */
+  --chakra-colors-gray-50: #00f;
+}
+```
```

Comment:
Is there a link to a list of all these theme-able variables? Ideally in the fidesui docs, but I'll take Chakra as a close second...

Reply:
The recommended approach would be to use the browser's inspector to find the variable you want to override. There isn't a static list of variables, because they are automatically generated from the theme object.

Would adding a description of how to find the desired CSS variable to change be better here? If not, I can do a one-time generation of the current list of variables and include it – though they may change in the future.

Reply:
It's also not going to be clear from the list of variables which ones you'd want to override, as they are contextless tokens.